recipes/cusparselt/meta.yaml

```
@@ -0,0 +1,93 @@
+
+source:
+  url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt-linux-x86_64-{{ version }}.{{ patch_version }}.tar.gz   # [linux64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt-linux-ppc64le-{{ version }}.{{ patch_version }}.tar.gz  # [ppc64le and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt-linux-sbsa-{{ version }}.{{ patch_version }}.tar.gz  # [aarch64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt_{{ version }}.exe  # [win64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+
+
+  # TBD
+  sha256: 63a5a854b2df4477ae48284a23c1b7c7632c23fe418434fa70a2ff8ac238b27e  # [linux64]
+  # sha256: d914a721b8a6bbfbf4f2bdea3bb51775e5df39abc383d415b3b06bbde2a47e6e  # [ppc64le]
+  # sha256: 031b75946d2ce22f8c2736f7b4216b9377406afcb35012093a66e2801bfe2990  # [aarch64]
+  # sha256: 88d3e2c662e601214ef5caa26ca2db0492eab6ec3bf9dc9a746bb1a7b8c3aab2  # [win64]
+
+build:
+  number: 0
+  skip: True  # [win32 or cuda_compiler_version in ("None", "9.2", "10.0")]
+  script:
+    - mkdir -p $PREFIX/include                                                # [unix]
+    - cp -r include/* $PREFIX/include/                                        # [unix]
+    - mkdir -p $PREFIX/lib64                                                  # [unix]
+    - mv lib64/{{ cuda_compiler_version }}/libcusparseLt.so* $PREFIX/lib64/   # [unix]
+
+    # - mkdir %LIBRARY_INC%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt.h %LIBRARY_INC%\\                               # [win64]
+    # - mkdir %LIBRARY_INC%\\cusparseLt                                                                                     # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\include\\cusparseLt\\types.h %LIBRARY_INC%\\cusparseLt              # [win64]
+    # - mkdir %LIBRARY_BIN%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.dll %LIBRARY_BIN%\\  # [win64]
+    # - mkdir %LIBRARY_LIB%                                                                                                 # [win64]
+    # - copy %SRC_DIR%\\libcusparse_lt\\libcusparse_lt\\lib64\\{{ cuda_compiler_version }}\\cusparseLt.lib %LIBRARY_LIB%\\  # [win64]
+  ignore_run_exports_from:
+    - {{ compiler('c') }}    # [linux]
+  run_exports:
+    - {{ pin_subpackage('cusparseLt') }}
```

One more thing: From the version it looks like cusparseLt is not yet stable (for backward compatibility etc) enough to obey versioning semantics? If it's the case, then it's best to pin it exactly:  
```suggestion
    - {{ pin_subpackage('cusparseLt', exact=True) }}
```

---

okay!